core/base/src/TColor.cxx

```diff
@@ -1980,6 +1980,27 @@ Int_t TColor::GetColor(Int_t r, Int_t g, Int_t b, Float_t a)
    return color->GetNumber();
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Static method returning color number for a color specified by name.
+///
+/// If specified color name does not exist -1 is returned.
+
+Int_t TColor::GetColorByName(const char *colorname)
```

Comment:
```suggestion
Int_t TColor::GetColorByName(const std::string &colorName)
```

Reply:
Other/Older TColor methods are using const char * (GetColor for instance). I think it is better to keep consistency... otherwise, we will need to change them all and we risk backward incompatibility.